service/project.go

```diff
@@ -548,10 +548,8 @@ func (uis *UIServer) setRevision(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// run the repotracker for the project
-	ts := util.RoundPartOfHour(5).Format("2006-01-02.15-04-05")
-	j := units.NewRepotrackerJob(fmt.Sprintf("catchup-%s", ts), projectRef.Identifier)
-	err = evergreen.GetEnvironment().RemoteQueue().Put(j)
-	if err != nil {
+	j := units.NewRepotrackerJob(fmt.Sprintf("ui-triggered-job-%d", job.GetNumber()), projectRef.Identifier)
```

Comment:
job.getNumber is monotonically increasing, but the source of the number is in the application process so it resets on restarts and is duplicated on each application server.

Reply:
Fixed.